configure.ac

```diff
@@ -44,7 +44,7 @@ dnl *****************************************
 LIBGNOME_REQUIRED=2.0.0
 LIBGNOMEUI_REQUIRED=2.0.3
 GLIB_REQUIRED=2.14
-GTK_REQUIRED=2.12
+GTK_REQUIRED=2.24
```

Comment:
@markuspg Increasing GTK requirements seems like a no-brainer.

Probably we actually require newer glib too. [GTK 2.24 requires glib 2.28](https://github.com/Unity-Technologies/gtk/blob/gtk-2-24/configure.in#L35), but Ubuntu 14.04 contains glib 2.40, so maybe we should just go for that.

Thoughts?

Reply:
Honestly I'm tempted to drop _Ubuntu 14.04_ as baseline. Recently I corrupted the VM and upon reinstalling it I ran into issues with package certificates. Right now I develop on _Fedora 37_ where I installed `qof` manually from sources. The only downside is that I have to uncomment any _GtkHTML_ references because I cannot compile it on recent Fedora.